dia/diameter_3gpp_ts29_061_sgi.dia

```diff
@@ -421,6 +423,9 @@
               [ 3GPP-Negotiated-DSCP ]
               [ TWAN-Identifier ]
               [ 3GPP-User-Location-Info-Time ]
+              [ TP-NAT-IP-Address ]
```

Comment:
the pool id should IMHO go into here as well

Reply:
updated